chainer/functions/connection/n_step_rnn.py

```diff
@@ -144,10 +144,22 @@ def get_random_state():
         libcudnn.CUDNN_LSTM: True,
     }
 
+_rnn_persistent_algo = None
+if cuda.cudnn_enabled and _cudnn_version >= 6000:
+    _rnn_persistent_algo = {
+        'standard': libcudnn.CUDNN_RNN_ALGO_STANDARD,
+        'static': libcudnn.CUDNN_RNN_ALGO_PERSIST_STATIC,
+        'dynamic': libcudnn.CUDNN_RNN_ALGO_PERSIST_DYNAMIC,
+    }
```

Comment:
How a user select 'static' or 'dynamic'? What's the difference?

Reply:
'static' only supports small network, but 'dynamic' supports both small and large networks.
'dynamic' need to call `create_rnn_persistent_rnn_plan `.  This function needs computation cost. So we need to reuse it.